src/lib/organisms/WerkvormCard.svelte

```diff
@@ -97,6 +98,9 @@
 		font-size: 1.4rem;
 		font-weight: 700;
 		margin-bottom: var(--unit-small);
+		white-space: nowrap; 
+        overflow: hidden; 
```

Comment:
inspringen!

Reply:
In mijn vscode is deze tab er niet, ik denk dat er tijdens het pushen wat is gebeurd.